tests/dummy/app/routes/nodes/detail/draft-registrations/index.js

```diff
@@ -8,6 +8,8 @@ export default Ember.Route.extend({
     },
     setupController(controller, model) {
         this._super(controller, model);
+        var metaschemas = this.store.findAll('metaschema');
+        controller.set('metaschemas', metaschemas);
```

Comment:
Are there any concerns about, eg, needing to wait on the resolution of the promise?

(certainly for a computed; I'm less sure about what happens you you set a promise as a property on the controller directly)

I couldn't examine this locally,  so instead I'll just ask you. I received an error response with a 200 status code from the API for a project with 1 draft and one completed registration. The API issue is unrelated to this PR. (pinged @briang for varnish triage)
```
Request URL:https://staging-api.osf.io/v2/nodes/8k3w2/draft_registrations/
Request Method:GET
Status Code:200 OK

{"data":[{"errors": [{"detail": "Not found."}]}],"links":{"first":null,"last":null,"prev":null,"next":null,"meta":{"total":1,"per_page":10}}}
```